src/lib/resources/record.resource.ts

```diff
@@ -129,4 +129,14 @@ export class RecordResource extends BaseResource {
 
     return this.getVoFromResponse<RecordResponse>(response, 'RecordVO');
   }
+
+  public async getRecordById(recordId: number): Promise<RecordVO> {
```

Comment:
Do we want test coverage for this? I actually don't know if we do or not.

Reply:
I think it's a good idea!